src/queue.ts

```diff
@@ -1124,10 +1128,28 @@ const queue = {
             "%)",
           ""
         );
-      result += "...";
-      result += weightedList.entries.length > 5 ? "etc." : "";
-      result += " (" + weightedList.offlineLength + " offline)";
-      return result;
+
+      if (weightedList.entries.length > 5) {
+        onlineList += "etc.";
```

Comment:
I would add a translation for this word. For example `etc.` is `usw.` in German.
Maybe we could even give the `onlineList` as the argument of the translation and then have an etc. version and a version without etc.?